packages/terra-application-docs/src/dev-site-extended-test/component/ThemedComponent.jsx

```diff
@@ -1,7 +1,9 @@
+/* eslint-disable import/no-unresolved */
```

Comment:
Just wondering, which import is causing the need for this? None of them are standing out to me as different from the other imports in this PR

Reply:
it's both the imports
```js
import { ApplicationIntlContext } from 'terra-application/application-intl';
import { ThemeContext } from 'terra-application/theme';
```
It's weird because they are working in other packages and webpack doesn't complain. If I change the name to something nonexistent like `terra-application/theme-test`, webpack gives an error so the imports do seem to be working correctly. So I'm not sure why the linter is failing.